docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.profile

```diff
@@ -173,7 +173,6 @@ function su_humsci_profile_preprocess_image_formatter(&$variables) {
   if (isset($variables['url'])) {
     // Disable screen readers from seeing the link on the image since there
     // should be another link with text nearby.
-    $variables['image']['#alt'] = '';
```

Comment:
@hectorlj Good find! However, the comments on this code make it seem intentional. I think we may also need to add `aria-hidden="true"` so screen readers truely skip over the image.

Reply:
Fixed for both preprocessor functions.